config/rbac/role.yaml

```diff
@@ -44,6 +44,17 @@ rules:
   - list
   - patch
   - update
+- apiGroups:
+  - apps
+  resources:
+  - daemonsets
+  verbs:
+  - delete
```

Comment:
can you only set get list and watch ?

we dont need patch update and delete

Reply:
Done